CHANGELOG.md

```diff
@@ -22,7 +22,7 @@ _None._
 
 ### Bug Fixes
 
-- Fix the `annotate_test_failures` to include test cases with error nodes in the failures list. [#58]
+_None._
```

Comment:
This accidentally went into the template, not the the visible `## Unreleased` section

Reply:
Good catch!